### ISSUE TITLE
Stronghold Crusader Balance Patch

### ISSUE THREAD

**Issue by LordJaeger** [UnofficialCrusaderPatch/UnofficialCrusaderPatch#962]
As most people know, Stronghold has some serious balance issues which stunts its potential in multiplayer competetive playing (and makes singleplayer less challenging as the ai does not use certain tacticts etc). Some units are overpowered, like siege units (shields, catapults), crossbowmen, horse archers etc; but also other aspects of the game is unbalanced, such as the ability to build a monster economy, build 50 towers etc, which makes sieging both unrealistic and at times impossible. So, a few easy changes can be made to make the game better and more fun. Here are some ideas to change:

Economy:

We all know there is a problem with the way the economy works in this game. Especially against the ai, it becomes easier the longer the game lasts. 

-So each production unit should have a cap on production. Stone quarries as an example, could have a 100 % efficiency rate at 2 units, meaning, at 3, all 3 stone quarries produce at 75 % efficiency, and at 50 % efficiency if having 4 stone quarries. So having 4 stone quarries would be no better than 2. Iron mines have the same efficiency rating. A similar system should be applied to all forms of production or at least on some of them. Altnernatively, the price should be reduced, so as to make it more difficult to build the economy in mid-late game. This would prevent the economy becoming unrealistically huge. Moreover, some buildings, if not all, could also scale in cost: first iron mine= 20 wood, second = 40, third = 80 etc. This would make “economic victories” more important in the game, when destroying an iron mine etc. In general, this would make it useless to have 100 bakeries etc, and the player would be forced to play more like the ai does, as the ai never expands after a certain limit. 

Castle building:

-The distance for castle building and keep needs to be reduced. This would prevent these chinese wall type strategies making it impossible to siege.

-Moreover, the same scale system as earlier described, should also be used for towers and gate houses. If a tower cost 40 stone, the second tower should cost 80, the third: 160, the fourth: 320, the fifth: 640, the sixth: 1280 etc. 

-Also towers should not be able to be repaired, or if they can, then it should take time for this repair to occur. All in all, this would make taking a tower down in it self a victory and it would prevent the building of many towers.

-Moats should cost money to dig, soldiers should, if the moat is to wide, be able to walk through it, or a limit should be placed on how wide a moat can be. (just an idea)

Army:

-Siege engines should scale in cost in the same system as earlier described. This would make it difficult or insanely expensive to build many siege engines or shields. We all know how stupid and overpowered shields and catapults are...This would not apply to the price of engineers themselves.

-Moreover, catapults should have 20-30 % shorter range and should move at half the speed, while also taking more time to prepare for every focused move. The same idea should apply to ballistas. 

-Shields should have 50 % less health, and should be countered by slingers, ballistas and tower ballista. Alternatively, fire arrows could do more damage against them. 

-Both horse archers and crossbowmen should deal at least 50 % less damage; and they should both also have a reduced range to at least 10-30 % less than other archer units. Moreover they should also cost more. This would make them easier to counter. The game should be less archery and range based.

-Spearmen and monks should have 100 % more health against ranged units.

-European archers should deal 20-30 % more damage against armoured units.

-All fire should do half the amount of damage to all units. Fire is to overpowered. 

Some ideas that could be implemented in the long run.

**Comment by GRhin** [comment]
Alot of ideas here. And many will one day be available for you to do
yourself, so my response is purely personal feedback on the ideas, not
judgement or indicators of whether it will be available.
I like that you are thinking of balance and how to improve it, but as you
will see I think the basis for your assumptions may be a little off.

Limits to unit/building types: while this might make AIs more difficult,
and so may be worth looking into, it is treating a symptom of the problem
of AIs just not being intelligent enough. That's not going to be fixed any
time soon so working to the symptoms might be worth it, I just prefer to
move to multiplayer (which with a zero gold start feels alot more like
playing single player than the traditional peacetime games, there is a
community playing that with a custom balance out there, check Krarilotus's
YouTube streams for that) seeing as the ai will always be a weakpoint for
this game.


Reduced castle range: shorter walls are easier to put units on, I
personally don't think this would have much effect on the balance or
enjoyment of the game.

No tower repair: again, I don't see this as having much effect if any,
except to make the game less enjoyable. Tower repair is a huge quality of
life feature that should not be so easily dismissed.

Moats costing gold: the problem with moats is that AIs will never be good
at dealing with them. It is almost too strong to be available against AI.
On the other hand having to pay for it you may as well just put walls up
which will survive longer if it is brought under attack. This is like the
first point, treating a symptom (ineffectively in this case in my opinion)
of the ais lack of intelligence. So I prefer to move to multiplayer when
this becomes a problem.
In multiplayer the free moat is essential, and it's not really free,
because of the time it takes to dig, and the units lost as the enemy tries
to prevent you from digging (especially as they are usually dig by
unamoured slaves). This is from the perspective of the no gold start
mentioned earlier, even building walls is a risk as the stone is often
better sold to buy units. In the high gold peacetime games, adding a cost
to moat would make next to no difference.

Catapults slower and shorter range: okay unpopular opinion here, but
catapults are not as overpowered as most people think. They appear that way
because against ai by the time your considering building them you are so
far ahead economically that you can afford an atrocious amount of them.
Replacing them with trebs that have no speed they are still overpowered so
the speed change suggestion is unlikely to make much difference. Shorter
range is likely to have a huge effect, and I'd be willing to try it out
(though i don't think we have found how to effect range yet), just not the
the degree you are suggesting, as they are still countered pretty heavily
by mangonels assuming you can get a decent amount of them (which ai can't,
again symptom, resulting in a move to multiplayer)
I would prefer to reduce the firerate instead of the two suggestions you
made.

Sheilds: I think the custom multiplayer balance mentioned earlier reduces
HP, but that's after i last checked in.
I disagree with your counters suggestions, but that's because we have
already found another way of nerfing them that fits their historical
setting much better - make them slower, the aforementioned custom balance
made them the same speed as pikemen, which means they are countered by
melee units, particularly macemen and knights. Doing this also nerfed horse
archers as used in multiplayer as before the ha and sheilds moved at almost
the same speed allowing the ha to always be protected.

Ha and xbow: gotta hard disagree with the damage nerf. Xbow are already
marginal for gold cost against most units compared to European units. As
for ha, they are another unit that appears strong more due to the fact that
you amass so many against AIs, you are nearly so scared of the ai using
them however.
That being said ranged units in general could do with a nerf or melee units
a buff. The aforementioned custom balance I believe does both.

Spears and monks increased ranged resistance: hard disagree once again,
particularly for the monks. That is their only downside, increase that
resistance and they become a super unit that's almost free! Monks are in a
really good position as an anti melee unit. They outperform every other
melee unit in hand to hand combat, if you see your enemy with them, you
have to fight with ranged units, fighting any other way is far too
expensive.

Euro archers do more damage vs armoured units: that will basically result
in xbow being pointless. As mentioned earlier euro archers are more cost
effective against almost every unit, the only exceptions being the units
you want to buff them against. Hard disagree.

Fire: don't let Kimberly hear you saying fire is OP you will be insta
banned so hard your great great grandkids won't be able to comment
In all seriousness, fire is strong, but not overpowered,because all
implementations of fire are incredibly vulnerable. Firethrowers and boiling
pots are shot by archers before they can be used, pitch can be prevented by
destroying/clearing walls with catapults, or by letting a few cheap units
trigger them. Fireballista has to burn buildings to get fire on units,
which are usually behind walls, which should have ballista or mangos on
them to destroy the fire ballista, extra walls can be built to block arrows
from specific buildings, and in emergencies the building can be destroyed
before being lit and rebuilt. Ai can't do these, so again maybe another
symptom being treated.

In summary I like how you are thinking but many of these ideas are trying
to force the player to play at the AIs low level rather than trying to
improve the balance. Those that are specifically addressing the balance
tend to ignore the actual details of the balance, probably due to your
personal play style emphasizing specific units or being weak with others.
If you can get in on Krarilotus's multiplayer games you will probably get a
much more accurate feel for the balance (admittedly his custom balance) and
may help you understand some of why I disagree with so much of your
suggestions.

**Comment by LordJaeger** [comment]
To GRhin

Thanks for the reply! I think what i forgot to mention, is that Stronghold is more balanced than people think, but even in real life, we have certain nerfs, in real warfare, like with nukes, napalm etc, because they are dishonorable and well unbalanced. I mean a world war with nukes would be no fun at all, even though there are counters to them, if you understand? 

The problem with stronghold is that it has to much freedom. I mean, in what other game can you build so many different siege equipments, or turn land into water with moats, or build towers with siege equipments on, or combos with shields and units or fire that will actually damage everything except walls, fear factor etc. Because it has so much freedom and material (which i like), it does bring with it many complications. The reason other games (like AoE for example), are more balanced, is because there are more restrictions in the game for things you can do. And even in such games, there are changes made all the time. 

And yes, I know that technically speaking there are counters to everything, but honestly is it fun with a shit fest of 50 towers and horsearchers and shields and 100 catapults? I have been playing online and it is even LESS fun than with the Ai. I am not saying that all of these changes should be made or to that degree, I am just highlighting some issues with the game. We respectfully disagree on some issues here and that is fine. And I am not treating a symptom here, the cause is simply to few restrictions.

-Monks as an example, yes they are cheap, but with proper defence, they fall like flies, so perhaps it would work to give them 100 % more health against archers, but increase cost to 15 or 20? Just an idea.

-And trebuchets become useless if the catapults are cheeper and can move. I like the trebuchets because the catch is, you cant move them. That is one example of a restriction the game does well, and as a result they are not over powered.

-Reduced castle range i think would have a HUGE effect, because it would force people to actually build castles, as you would have to protect the keep. And again, the game does have a restriction on the range already, having no restriction on this part would be a disaster as all would agree on. Why not reduce it a bit more?

-On repair, i like it, but, i have been playing online where i pound a castle and sacrifice many resources doing so, and the other player simply repairs and repairs, and the tower never goes down. It is neither fun nor realistic (not saying that realism is always good, but you understand), if the enemy then has a huge supply of stone, well then the tower is never going down so what is the point? Perhaps one could be able to repair, but it would cost two or three times the amount? It would be a catch, and maybe worth it during a siege or maybe not? 

-Making shields slower I agree with, and a reduction in HP i still stand by, they can take a ridiculous amount of arrows and cost almost nothing.

-On moats, perhaps you are correct

-Crossbows are still OP, the same with HA. I think a good game should have good counters and different strategies. Example: Euro archers should have longer range than HA and CB, shoot faster of course and would be better against monks and spearmen. And perhaps increase their cost. But take knights aginst Euro archers, 167 shots, is that really balanced? Crossbows do more damage, but should have shorter range. Horse archers counter siege units and crossbows, but if they have shorter range, then they again are countered by Euro og arabian archers. Crossbows should do less damage against all other units than horsearchers and siege or something to that effect.

-I stand by what I said on the economy, at a certain point in the late game, the amount of resources is just to much. Either do what I explained or alternatively make a limit on how many buildings of the different types you can build. Again, just an idea. 


An important thing to note is this: I think the developers wanted us to play like the AI. They never intended people to use 100 shields with HA etc. So they should have then forced us to play in such a way or in a similar way. If the AI was changed to the point where they would play like most people, it would be no more fun in my opinion. I dont think it is unfair to say that the game was (when released) unfinished considering the balance and it would be the correct measure to see some of the issues fixed. But, it must be mentioned that I dont think that all the changes I listed would work, I am just stating some balancing problems and some ideas that MIGHT help correct them.
